marshmallow_utils/fields/localizededtfdatestring.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Localized Extended Date(/Time) Format Level 0 date string field."""
+
+import calendar
+from datetime import date, datetime
+from edtf.parser.grammar import Interval, ParseException, level0Expression
+from marshmallow import fields
+
+
+def _(x):
+    return x
+
+# TODO: move to external package and depend on flask-babelex
+
+class LocalizedEDTFFormatter(object):
+    """Localized Formatter for EDTF dates."""
+    RANGE_SEPARATOR = u' \u2014 '
```

Comment:
Why is `\u2014` used and not `-` directly?

Reply:
That's my request :-)